mpt_trie/src/debug_tools/stats.rs

```diff
@@ -370,8 +370,8 @@ mod tests {
         let hash_entries = generate_n_random_fixed_trie_hash_entries(n_hash_nodes, seed + 1);
 
         let mut trie = HashedPartialTrie::default();
-        trie.extend(val_entries);
-        trie.extend(hash_entries);
```

Comment:
Are we not better `?`ing the error up to let the caller do what they need? `extend` only gives us `Ok` so it's not particularly useful but it's probably better than panicking.

Reply:
in the test, right, because this is in a test

Reply:
So it is. Ignore me. Apologies. The lack of `#[test]` made me miss that